pkg/test/main/testmain.go

```diff
@@ -77,7 +77,9 @@ func TestMainSetupMultipleEnvironments(m *testing.M, testType test.TestType, crd
 		mgr, stop := testcontroller.StartTestManagerInstance(env, testType, whCfgs)
 		stops = append(stops, stop)
 
-		testcontroller.EnsureNamespaceExists(mgr.GetClient(), k8s.SystemNamespace)
+		if err := testcontroller.EnsureNamespaceExists(mgr.GetClient(), k8s.SystemNamespace); err != nil {
+			log.Fatalf("error ensuring namesapce exists: %v", err)
```

Comment:
We should add a common-typo linter at some stage :-)

Reply:
oh it's coming hehe